xml/ha_storage_protection.xml

```diff
@@ -781,18 +780,42 @@ Timeout (msgwait)  : 10
        would be appropriate if the <literal>msgwait</literal> timeout value for
        SBD was set to <literal>30</literal> seconds.</para>
      </callout>
-     <callout arearefs="co.ha.sbd.st.rsc">
-      <para>
-       The &stonith; resource to use for SBD.</para>
-     </callout>
-     <callout arearefs="co.ha.sbd.pm_delay_max">
+   </calloutlist>
+  </step>
+  <step>
+   <para>For a two-node cluster, decide if you want predictable or random delays. For
+   other cluster setups you do not need to set this parameter.</para>
+   <variablelist>
+    <varlistentry>
+     <term>Predictable Static Delays</term>
+     <listitem>
+      <para>This parameter enables a static delay before executing &stonith; actions.
+      It ensures that the nodes do not fence each other if separate fencing resources
+      and different delay values are being used. The node with the shorter delay will
+      loose in a <quote>fencing race</quote>.
+      The parameter can be used to <quote>mark</quote> a specific node to survive
+       in case of a split-brain scenario in a two-node cluster.
+      <remark>toms 2018-05-09: From FATE#324488, hmn, not sure how to do that:
+      "They need to configure two stonith resources respectively targeting the two nodes,
+      and adjust "pcmk_delay_base" parameters whenever a SAP master is promoted."</remark>
```

Comment:
Thanks for bringing this up and good question. This hasn't been decided yet though ;-) 

There's further discussion about automatic adjusting of pcmk_delay_base from here:
https://bugzilla.suse.com/show_bug.cgi?id=1074039

I'll let you know once we have a solution on this.